src/main/resources/com/owlcyberdefense/syslog/xsd/syslog-rfc5424.dfdl.xsd

```diff
@@ -204,7 +204,7 @@ SOFTWARE.
       <xs:element name="Message" minOccurs="0" dfdl:initiator="%SP;">
         <xs:complexType>
           <xs:choice>
-            <xs:element name="UTF8" dfdl:initiator="%#xEF;%#xBB;%#xBF;" type="sl:msgStr" dfdl:encoding="UTF-8" />
+            <xs:element name="UTF8" dfdl:initiator="%#xFEFF;" type="sl:msgStr" dfdl:encoding="UTF-8" />
```

Comment:
I suggest adding annotation-documentation to this UTF8 element explaining that this is a BOM character (0xFEFF) which will be the 3 bytes EF BB BF in UTF-8 encoding.